src/utils/utils.go

```diff
@@ -41,6 +41,18 @@ const (
 	W_ALL
 )
 
+// FIXME: Remove this code when bug 1968606 is fixed
+// https://github.com/openshift/ocs-operator/blob/master/controllers/util/status.go#L14-L32
+// These constants represent the Phase as used by SC.Status.Phase for OCS
+var (
+	// SCPhaseError is used when SetErrorCondition is called
+	SCPhaseError = "Error"
+	// SCPhaseReady is used when SetCompleteCondition is called
+	SCPhaseReady = "Ready"
+	// SCPhaseProgressing is used when SetProgressingCondition is called
+	SCPhaseProgressing = "Progressing"
+)
```

Comment:
Can't we reuse it from the `ocs-operator` code?

Reply:
Yes, done